apps/patient/src/views/Pharmacy.tsx

```diff
@@ -96,6 +96,15 @@ export const Pharmacy = () => {
     setShowingAllPharmacies(false);
   };
 
+  const showToastError = () =>
+    toast({
+      title: isReroute ? 'Unable to reroute order' : 'Unable to submit pharmacy selection',
+      description: isReroute
+        ? 'Your prescription has already been received. Text us if you need to update your pharmacy.'
```

Comment:
@jknapp25 @thebadprince could y'all look at this copy?

Reply:
Title: `Unable to change pharmacies`
description: `Your order is already being processed. Text us if you need it sent to a different pharmacy.`

I think this makes it vague enough (we're not saying their order is confirmed) but still more specific as to why they can't change pharmacies (order is being processed). There's also no internal lingo of "reroute" so it's more straight to the point. Lmk if that makes sense 🙏 

ps. Curious: what does the toast look like/how does it currently behave?

Reply:
<img width="356" alt="Screen Shot 2024-03-06 at 5 29 44 PM" src="https://github.com/Photon-Health/client/assets/700617/3e3a2f75-05be-4bef-ad3e-9e4bdff682be">

Reply:
tiny nit: i think we can use chakra's warning toast so this is orange instead of red. i prefer to use error if the action is destructive or there's a system error